maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderEquityController.java

```diff
@@ -47,6 +57,7 @@ public ResponseEntity<Void> createRepOrderEquity(@RequestBody RepOrderEquityEnti
     @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE))
     @ApiResponse(responseCode = "404", description = "Not Found.", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ErrorDTO.class)))
     public ResponseEntity<Void> createRepOrderEquity(@PathVariable Integer id, @RequestBody RepOrderEquityEntity repOrderEquityEntity) {
+        LoggingData.MAAT_ID.putInMDC(id);
```

Comment:
Is the id what we want to store here? It appears the id of the RepOrderEquity object is not the same as a Rep Order ID, so perhaps doing what you did in the method above (repOrderEquityEntity.getRepId()) is a better option?

Unless I misunderstand what is happening here, in which case ignore me :)

Reply:
Good spot! You are correct,have amended to:
`LoggingData.MAAT_ID.putInMDC(repOrderEquityEntity.getRepId());`

Reply:
I'll have another look through the remaining calls to putInMDC in this PR in case I missed any others :)